x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.js

```diff
@@ -53,6 +53,51 @@ class LinksMenuUI extends Component {
     };
   }
 
+  openInDiscover = async () => {
+    const {
+      services: { share },
+    } = this.props.kibana;
+    const discoverLocator = share.url.locators.get('DISCOVER_APP_LOCATOR');
+    const job = mlJobService.getJob(this.props.anomaly.jobId);
+
+    const index = job.datafeed_config.indices[0];
+    const dataViewId = (await getDataViewIdFromName(index)) || index;
+
+    const record = this.props.anomaly.source;
+    const from = timeFormatter(record.timestamp); // e.g. 2016-02-08T16:00:00.000Z
+    const to = timeFormatter(record.timestamp + record.bucket_span * 1000);
+
+    let luceneQuery = '';
```

Comment:
why Lucene but not KQL? 🤔

Reply:
Good point! I updated it to KQL now. Thanks.